provisioning/security/tpm-provider-emulator/src/main/java/com/microsoft/azure/sdk/iot/provisioning/security/hsm/SecurityProviderTPMEmulator.java

```diff
@@ -102,6 +105,20 @@ public SecurityProviderTPMEmulator(String registrationId) throws SecurityProvide
         srkPublic = createPersistentPrimary(tpm, SRK_PERSISTENT_HANDLE, TPM_RH.OWNER, SRK_TEMPLATE, "SRK");
     }
 
+    /**
+     * Constructor for creating a Security Provider on TPM Simulator with a cached nonce from the provisioning service
+     * Used for reconnecting via TPM only after device has been provisioned
```

Comment:
I like this explanation, but I'd like it even more if it linked to some sample code that demonstrates what you are describing. I think this scenario is worth writing a new, simple sample for. Thoughts?